docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -5,27 +5,38 @@ outline: deep
----
+* [概要](#概要)
+  
+* [統一内部フォーマット](#統一内部フォーマット)
+  
+* [プロトコル分離](#プロトコル分離)
+  
+* [属性タイプマッピングテーブル](#属性タイプマッピングテーブル)
+  
+* [システム属性の違い](#システム属性の違い)
+  
+* [出力フォーマットの違い](#output-format-differences)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)